dist/about.html

```diff
@@ -28,69 +28,72 @@
     </div>
     <div class="nav-container lato_regular">
-      <img class="logo" src="./img/logo/logo-black.png" alt="" />
+      <a class="menu-link" href="./index.html"><img class="logo" src="./img/logo/logo-black.png" alt=""/></a>
       <nav class="nav">
         <ul class="menu-nav">
           <li class="menu-nav__item">
             <a href="./about.html" class="menu-nav__link"> About </a>
           </li>
           <li class="menu-nav__item">
-            <a href="" class="menu-nav__link"> Program </a>
+            <a href="" class="menu-nav__link"> Series </a>
           </li>
           <li class="menu-nav__item">
             <a href="" class="menu-nav__link"> Join </a>
           </li>
           <li class="menu-nav__item">
-            <a href="" class="menu-nav__link"> Sponsor </a>
+            <a href="" class="menu-nav__link"> Tier List </a>
           </li>
           <li class="menu-nav__item">
             <a href="" class="menu-nav__link"> News </a>
           </li>
         </ul>
       </nav>
-      <a href="#" class="menu-nav__btn"> CC Campaign </a>
+      <a href="https://anilist.co/user/Zeraltz/" class="menu-nav__btn"> AniList </a>
     </div>
   </header>
   <main>
     <section class="about-hero lato_regular">
-      <p class="about-hero__quote">Hello! Sharing World!</p>
-      <h1 class="about-hero__title cocogoose_regular">CREATIVE COMMONS GLOBAL SUMMIT 2015</h1>
+      <p class="about-hero__quote">"Hello! Anime Fans!"</p>
+      <h1 class="about-hero__title cocogoose_regular">ANIMATION FANS ALL OVER THE WORLD! GATHER!</h1>
       <hr class="desktop about-divider">
-      <p class="about-hero__text-content">Lorem ipsum dolor, sit amet consectetur adipisicing elit. Cum nam distinctio, quaerat aspernatur quos iusto ipsa, voluptatem sunt aut, nulla sit accusantium magni itaque nesciunt unde dolorem minus possimus at deleniti optio dicta!</p>
-      <p class="about-hero__contact-us">Please contact us per Email for any further questions about CC Global Summit 2015 <span class="about-hero__contact-us__email">ccsummit2015@cckorea.org</span></p>
+      <p class="about-hero__text-content">I am a 27 seven years old half Hispanic, half Asian guy from South America, I have been watching anime consistently since 2009. In all these years I have watched more than 400 shows (adding up to 3100 hours) and I usually critique them based on facts, general consensus and my own feelings. I like writing code in my free time, and also writing novels (I'm bad at this one tho).
+      I speak English and Spanish, I tried to learn German by myself but I gave up, and now learning Chinese by myself as well.
+    
+      Right now I'm studying to become a Full Stack Web Developer, and this is one of my projects, THANK YOU FOR BEING PART OF IT</p>
+      <p class="about-hero__contact-us">Please contact me per Email for any further questions about Animation Fans 2022 <span class="about-hero__contact-us__email">info@animationfans.com</span></p>
```

Comment:
- [ ] The page text and description is overlapping and therefore not showing fully. Kindly fix this so that your design looks like the given design.

Your design
![image](https://user-images.githubusercontent.com/81932478/169599391-4651092c-fd2a-485c-b874-ac88f5d07824.png)


Given design
![image](https://user-images.githubusercontent.com/81932478/169599247-f884c200-9b01-44f9-83ad-8a7d8b01daf0.png)

Reply:
@Munsa1 
Could you please elaborate more on this topic or test it again?
I have tested this part on 6 different devices and it works properly on every single on (iPhone 11, Xiaomi Mi 9T, Benq Monitor Desktop 1920x1080, Dell Laptop 1366 x 768 )
![unnamed](https://user-images.githubusercontent.com/96924302/169610681-d44f3316-3469-407c-a19f-87232f033abf.jpg)
![unnamed (1)](https://user-images.githubusercontent.com/96924302/169610683-70c1bcd3-cfa1-44f1-a13a-ea2ba213a28f.jpg)
![IMG_0219](https://user-images.githubusercontent.com/96924302/169610684-03d4fbf7-0379-4313-bf8c-709daf18fbfb.jpg)
![IMG_0220](https://user-images.githubusercontent.com/96924302/169610687-8831bbf3-4489-4692-90f6-3f3cc42182d9.jpg)
![IMG_0221](https://user-images.githubusercontent.com/96924302/169610689-b64b46e1-582d-484f-8eee-860067c4d7c7.jpg)
e